get-started/guide-neo4j-browser/guide-neo4j-browser.adoc

```diff
@@ -55,11 +56,60 @@ You can provide your name and send us a message.
 
 == Neo4j Browser Sync
 
-Neo4j Browser Sync is a new feature that allows you to synchronize your favorites, styling and settings to a cloud storage, so you can share it across machines and browsers.
+Neo4j Browser Sync is a feature that allows you to synchronize your favorites, styling, and settings to cloud storage, so you can share it across machines and browsers.
+
+image::{img}neo4j-browser-sync.png[width=500]
+
+You just need to click the cloud icon in the left-side menu and log in with your GitHub, Twitter, or Google account and synchronize your workplace to the cloud.
+
+== Built-in Guides
+
+If you want to learn more about Neo4j, Cypher, or the Neo4j browser before diving in, click on any of the helpful links shown after startup (commands also shown below).
+These are quick guides that introduce the different concepts.
+You can find more helpful links in the left sidebar of Neo4j in the "Documents" tab (icon:book[]).
+
+[%autowidth.spread]
+|===
+|*Title* |*Description* |*Command*
```

Comment:
options=header in the table
you can also format the third column as monospace, I think with cols="a,a,m" 
https://asciidoctor.org/docs/user-manual/#tables

Reply:
Sorry, I didn't follow this one. Why do I need options=header and how do I put that in the table? Also, is monospace the same as the `syntax`?